diffsmhm/diff_stats/cuda/tests/test_sigma.py

```diff
@@ -11,6 +11,12 @@
 
 @pytest.mark.mpi_skip
 def test_sigma_serial_cuda():
+    try:
+        _ = cp.array([1])
+        qp = cp
+    except RuntimeError:
+        qp = np
+
```

Comment:
This can use the cupy convenience function, right?

Reply:
There is a convenience function for this, but on Polaris it's unreliable. Sometimes it works fine, returning False when there's no gpu device, but other times it throws an error about cuda backend failing when there's no device

Reply:
Oh wow. We use it in other parts of the code. Should we get rid of that usage then?